### ISSUE TITLE
Release failed due to 401 Bad credentials

### ISSUE THREAD

**Issue by suzuki-shunsuke** [sachaos/toggl#12]
The release workflow of v0.6.1 failed.

https://github.com/sachaos/toggl/releases/tag/v0.6.1

<img width="241" alt="image" src="https://github.com/sachaos/toggl/assets/13323303/9598c613-6d03-49d8-98c9-dc1ca717529a">

https://github.com/sachaos/toggl/actions/runs/8913348340/job/24478654808

```
  ⨯ release failed after 21s                 error=scm releases: failed to publish artifacts: could not release: POST https://api.github.com/repos/sachaos/toggl/releases: 401 Bad credentials []
Error: The process '/opt/hostedtoolcache/goreleaser-action/1.25.1/x64/goreleaser' failed with exit code 1
```

https://github.com/sachaos/toggl/blob/af2562bbfafe0ed6131edf2cd576476db195e914/.github/workflows/release.yml#L25-L26

Probably the secret `COMMITTER_TOKEN` is wrong.

**Comment by suzuki-shunsuke** [comment]
By updating the secret `COMMITTER_TOKEN` and rerunning [the workflow run](https://github.com/sachaos/toggl/actions/runs/8913348340/job/24478654808), you would be able to release v0.6.1.

**Comment by sachaos** [comment]
@suzuki-shunsuke Thank you. I'll update the credential.

**Comment by sachaos** [comment]
@suzuki-shunsuke I've updated the token and released v0.6.1.
thanks for letting me know!
